docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+services:
+  simple-jade-pin-server:
```

Comment:
```suggestion
  simplejadepinserver:
```
Avoid using `-` in YAML keys due to potential compatibility problems.

Reply:
Changed from `-` to `_`